.travis.yml

```diff
@@ -28,8 +34,10 @@ jobs:
         - rm -rf $HOME/.cargo/registry/index
 
     - name: "Standard tests + code coverage"
-      # https://github.com/travis-ci/travis-ci/issues/5175#issuecomment-215016943
-      language: bash
+      language: node_js
```

Comment:
what overhead does this add to build times?
why is it needed?

Reply:
Super hard to compare travis build times. this one is 7 minutes with my change: https://travis-ci.org/holochain/holochain-rust/builds/433737994, develop looks like most builds are 12 - 13 minutes...

I'm using a nodejs package as the target for my [Integration Test](https://github.com/holochain/holochain-rust/pull/163/files/86e1b3d23d0f774e827a84650495fc704f421b97#diff-a6f8a312ae4afe529bb9b0b217a00e06) ... i tried installing the apt nodejs package... but again, the old version included with trusty isn't viable.

Again, open to other suggestions... I'm no travis guru by any means : )

Reply:
it's this bit:

<img width="917" alt="screen shot 2018-09-27 at 9 01 41 am" src="https://user-images.githubusercontent.com/629710/46114003-fb4c4380-c233-11e8-9f86-ec9e78614d53.png">

so ~3 seconds. i think we can live with that @neonphog